src/index.html

```diff
@@ -13,6 +13,52 @@
     />
```

Comment:
HTML Formatting: The attributes of the `link` tag are not correctly formatted. When an HTML element has long attribute values or a number of attributes more than 2, start each one, including the first, on a new line with a 2-space indentation relative to the tag. The tag’s closing bracket should be on the same level as the opening one.